src/Resources/config/event_engine.yaml

```diff
@@ -27,6 +27,13 @@ services:
         tags:
             - console.command
 
+    event_engine.command.projections_create:
+        class: ADS\Bundle\EventEngineBundle\SymfonyCommand\EventEngineProjectionsCreateCommand
```

Comment:
Maybe we should move this to the projectors subfolder?

Reply:
Kleir